### PR TITLE
Add phase engine to capture tutorial

### PR DESCRIPTION
**Pull request by SatjaSivcev**

### CODE REVIEW

File: source/Camera/Basic/CaptureTutorial.md [hunk 1]
```diff
@@ -110,9 +110,10 @@ We may choose to configure settings manually. For more information about what ea
 We can create settings for a single capture ([go to source][settings-url]).
 ```cpp
 const auto settings = Zivid::Settings{
+    Zivid::Settings::Experimental::Engine::phase,
     Zivid::Settings::Acquisitions{
         Zivid::Settings::Acquisition{ Zivid::Settings::Acquisition::Aperture{ 5.66 },
-                                Zivid::Settings::Acquisition::ExposureTime{ std::chrono::microseconds{ 8333 } },
+                                Zivid::Settings::Acquisition::ExposureTime{ std::chrono::microseconds{ 8333 } } } },
```
Comment (chrisasc) [inline source/Camera/Basic/CaptureTutorial.md:116]: Did we miss the two } } before?

Comment (chrisasc) [reply source/Camera/Basic/CaptureTutorial.md:116]: If capture.cpp is the only place we use this setting, maybe we should have the same linting/structure as the actual sample

**Review by chrisasc — CHANGES_REQUESTED** [review]

File: source/Camera/Basic/CaptureTutorial.md (hunk 1 above)
Comment (SatjaSivcev) [reply source/Camera/Basic/CaptureTutorial.md:116]: We did miss it before.
Ok, I will merge the sample first and use the same structure here.

Comment (SatjaSivcev) [reply source/Camera/Basic/CaptureTutorial.md:116]: It is exactly the same already.

**Review by chrisasc — APPROVED** [review]
lgtm

**Pull request merged**